osgi-support/commercetools-tests-fragment/pom.xml

```diff
@@ -110,7 +110,9 @@
                             filter:="(osgi.extender=osgi.serviceloader.processor)"
                         </Require-Capability>
                         <Provide-Capability>
-                            osgi.serviceloader
+                            osgi.serviceloader,
+                            osgi.serviceloader;
```

Comment:
This line is duplicated now, is that really intended here?

Reply:
Yep this should be removed 👍